fluent-pseudo/src/lib.rs

```diff
@@ -56,6 +56,11 @@ pub fn transform_dom(s: &str, flipped: bool, elongate: bool) -> Cow<str> {
     let result_range = pos + diff..result.len();
     let transform_sub = transform(&s[range], flipped, elongate);
     result.to_mut().replace_range(result_range, &transform_sub);
+
+    if with_markers {
+        return Cow::from("[") + result + Cow::from("]")
```

Comment:
This is the first time I write Rust code. I assume there's more elegant way to do that?

Reply:
```suggestion
        return Cow::from("[") + result + "]"
```
I had a look through the Cow documentation, and nothing really jumped out at me. @nordzilla Is there a nicer way of handling this?